### ISSUE TITLE
Chrome extension stop working with Amazon Music Player

### ISSUE THREAD

**Issue by X-Cite86** [david-sabata/web-scrobbler#835]
The extension stop working with chrome 46.0.2490.86 m.
It say "play some music to get started", when I start the music you can  briefly see a magnifying glass and then shows the normal logo with "play some music to get started" again.
With Google Play Music the extension still works great, seems to be a problem with amazon music player
Extension version: 1.54

**Comment by greedoe** [comment]
it works for me in Amazon to a point. the song shows up in my last.fm profile page WHILE i'm listening to it. once the song is done, it's gone from my list as if i never listened to it. kinda defeats the idea of scrobbling.

**Comment by brarig219** [comment]
Still not working for me at all.  The red Last.FM icon just appears in the top corner of the browser window, but never changes to the green checkmark, to indicate scrobbling.

Once in a while, the grey "question mark" box will appear, as if the app is trying to figure out what song is playing.  Even if you click on it and input the artist and song information and click "ok", the box just goes back to the red Last.FM icon.  No scrobbling.  

Thanks to anyone who is working on this!

**Comment by X-Cite86** [comment]
Keep Calm. @Paszt fixed it with pullrequest #838 (https://github.com/david-sabata/web-scrobbler/pull/838). Just wait for the next extension update, I think this will fix this Problem if pullrequest is merged into master.

**Comment by brarig219** [comment]
Thanks for the update @X-Cite86 

No worries at all. I am thankful that those of you with the knowledge of how to fix these issues can work on it, because I would be of very little help.  Thanks again!

**Comment by MartinL0815** [comment]
My Chrome installation does still not scrobble on Amazon (.de).

So - even not really worrying - any chance to get this fixed?
